uniprot-config/src/main/resources/uniprot-fields.json

```diff
@@ -30,7 +57,8 @@
     "dataType": "string",
     "example":  "sample example",
     "fieldType": "field",
-    "description" : "this is a dummy description"
+    "description" : "this is a dummy description",
```

Comment:
should we remove all the dummy descriptions, or do you think it adds value?

Reply:
We need to add actual description. Can we leave them until i start consuming in swagger?

Reply:
sure